src/kube-runtime/src/init

```diff
@@ -130,7 +130,12 @@ else
   # priority=0
   # We should use k8s client to communicate api server, need to change this when RBAC is enabled
   CHILD_PROCESS="FRAMEWORK_RETRIEVER"
-  curl -s $KUBE_APISERVER_ADDRESS/apis/frameworkcontroller.microsoft.com/v1/namespaces/default/frameworks/$FC_FRAMEWORK_NAME > framework.json
+  if [[ -z $KUBE_APISERVER_ADDRESS ]]; then
+    KUBE_APISERVER_ADDRESS="https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT"
+    set -- -k -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)"
+  fi
+  curl -s "$@" \
```

Comment:
Please test this. runtime not use Bourne shell. It use busybox default shell

Reply:
it's Bourne shell https://github.com/microsoft/pai/blob/master/src/kube-runtime/src/init#L1